apps/website/src/pages/docs/components.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Components
+description:
+layout: ./_layout.astro
+propsPath:
+group: main
+---
+
+import ComponentsPage from '~/components/ComponentsPage.astro';
+
+<ComponentsPage />
```

Comment:
i think there is kinda no need to create this mdx file. most of the frontmatter properties are not even being used or are turned into exceptions.

you can name it `components.astro` and move all the code inside here. only thing you would need to add the wrapping layout yourself.

```astro
---
import Layout from './_layout.astro';
---

<Layout content={{ title: 'Components' }}>
  // ...
</Layout>
```

Reply:
Moved to astro file